scripts/upload.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+cd ..
+# test
+printf "\033[1;35mtesting...\033[0m\n"
+if python setup.py test
+then
+    printf "\033[1;35mtesting successful\033[0m\n"
+else
+    printf "\033[1;35mtests failed---aborting\033[0m\n"
+    exit 1
+fi
+# build
+printf "\033[1;35mbuilding...\033[0m\n"
+python setup.py sdist
+python setup.py bdist_wheel --universal
+# upload
+printf "\033[1;35muploading...\033[0m\n"
+while true; do
+        read -p "upload to PyPI? [y/n]: " yn
```

Comment:
I typically like these scripts to have a default (indicated with a capital letter in the prompt) so that the most common action is just an enter press away.

Now, if you want to make sure that uploads are intentional things, no default is okay